### DISCUSSION TITLE
Where to create issues about GitHub itself?

### DISCUSSION THREAD

**Discussion by MNF** [community/community#21882] [New to GitHub]
<p>I want to submit a small feature request about github site. I expected to find <a href="http://GitHub.com/GitHub" class="inline-onebox-loading" rel="nofollow noopener">GitHub.com/GitHub</a> or similar and to add a new issue to Issues tab. Bu there is no such page. Are you not dogfooding?<br>
What is a correct place to post suggestions about GitHub web site?<br>
In particular, I want to suggest to add “an<br>
ability to change width of file name column”.</p>
<p>E.g. I am unable to distinguish names of the folders<br>
<a href="https://github.com/aspnet/Security/tree/dev/src" rel="nofollow noopener">https://github.com/aspnet/Security/tree/dev/src</a>  , and I am not interested in latest commit details, so I should be able to change width of file name column.</p>

**Comment by lee-dohm** [comment]
<p>Thanks for this feedback!</p>
<p>We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>You can distinguish the names of the folders by hovering over them. A tooltip will show the full name of the folder.</p>

**Comment by MNF** [comment]
<ol>
<li>So is this forum  the best place to post suggestions about GitHub web site?</li>
</ol>
<p>2.I am often using IPad, and tooltip is not an option for touch devices.</p>
<ol start="3">
<li>I  missed your reply,  this forum by default doesn’t send notifications when someone reply to your question.</li>
</ol>
<p>To understand why, I’ve checked, that in my profile “Don’t send me any community emails” is unticked. Then I found that there is separate section “Notification Settings”, that has unticked “Automatically subscribe me to all topics or reviews I participate in”.</p>
<p>Related settings are located in different sections, which is confusing.And default to not receive notifications for my own questions is very unintuitive.</p>
<blockquote>
<blockquote>
<hr>
</blockquote>
</blockquote>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/mnf">@mnf</a></p>
<p>&gt; 1. So is this forum  the best place to post suggestions about GitHub web site?</p>
<p>We actually recommend you reach out to us via our <a href="https://github.com/contact" rel="nofollow noopener">contact form</a> for feature requests and feedback about <a href="http://github.com" rel="nofollow noopener">github.com</a>.</p>
<p>Thank you for the feedback about notifications for the forum and settings. We very much appreciate this input and will definitely consider it in our future iterations of this forum. </p>

**Comment by MNF** [comment]
<p>Sorry, but contact form link <a href="/git.io/c">https://github.community/git.io/c</a>  is broken.</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p><a class="mention" href="/mnf">@mnf</a> Apologies; that’s the incorrect link. The link should have been:</p>
<p><a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a></p>
<p>That should work better for you!</p>

**Comment by MNF** [comment]
<p>The Contact page tells that the form should be used if I have some sensitive data to post. My question is about feedbacks/suggestions that I happy to make public, to give other people opportunity to  vote and/or improve the suggestions.</p>
<p>Are you still recommending  to use  Contact page or there is a better channel?</p>

**Comment by nadiajoyce** [comment]
<p>At this time, the contact page is the best place for sharing feedback and suggestions for GitHub.</p>

**Comment by rubyFeedback** [comment]
I tried the contact page, but I found the UI distracting.

It would be so much cooler if github itself could have issue trackers, the same ones many of us use daily.
That is, for us to suggest things to the github team itself, ABOUT github (such as the github issue
tracker in general).

For instance, I wanted to suggest that github adds an "auto-close this issue" feature, so that I can
add a time-limit to the issues I open. Often after a few months I forget which issues I have open
(sorry, I am totally chaotic in my organisation), which would be useful. But I don't feel like using
the "contact" page ... it seems too cumbersome compared to the simple issue tracker github usually
uses.

**Comment by marsrobertson** [comment]
### Direct link: https://support.github.com/contact/bug-report

**Comment by TurboGhoster** [comment]
I would like to thank the team for two changes that I recommended in your survey: 1. Enabling edit session names and 2. linking LLM models to the session. You also enhanced session controls on deleting/achieving individual session and got rid of nuke all sessions button, so that covered all my pet peeves. Big Thumbs up!!!
